R/ensure-tpch-source.R

```diff
@@ -4,6 +4,16 @@
 #' @export
 tpch_tables <- c("customer", "lineitem", "nation", "orders", "part", "partsupp", "region", "supplier")
 
+#' Generate tpch data
+#'
+#' Generate tpch data at a given scale factor. By default,
+#' `data` is assumed to be relative to the current working directory, but
+#' you can set the environment variable `ARROWBENCH_DATA_DIR` to point to another
+#' (permanent) base directory.
```

Comment:
We should probably also mention something about the duckdb installation stuff that will ensue when running this, just so people don't freak out that there's compilations and stuff.

Also, what do you mean by "permanent" here?

Reply:
> Also, what do you mean by "permanent" here?

It is from here: https://github.com/voltrondata-labs/arrowbench#known-data-sources-and-versions

To me it indicates that once you set it, all generated data with end up there. I will modify.

Reply:
Huh, not a total surprise that I forgot that I (probably) wrote that or what I was intending at the time, but it strikes me as odd now 🙈. Thanks for removing it here (we could remove it form the readme too, though it's not an absolute requirement! that is bumping scope a bit)